_posts/2020-12-24-newpipe-0.20.7.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: post
+title: "NewPipe 0.20.7 released: YouTube Mix support, bug fixes and many improvements"
+short: "NewPipe 0.20.7 released"
+date: 2020-12-24 19:00:00
+categories: [pinned, release]
+author: Poolitzer
+image: newpipe
+excerpt_separator: <!-- more -->
+---
+
+Merry christmas, if you celebrate that, and almost happy new year, if you use a calender at least close to the gregorian one. We want to present you a little gift as a good way to start into 2021 AD: The next release of NewPipe. It comes with a lot of improvement, fixes, and even a big new feature. So let us dive into this release post
+
+<!-- more -->
+
+## New in this version
+YouTube mixes are a feature now, and they are dropping in hot (and with a lot of bass). Opening a mix in previous versions resulted in the first video of the mix playing, nothing else. Now you see the full playlist and even a nice radio icon as an indicator. Enjoy listing to never ending lists and thank [@Stypox](Stypox) as well as [@XiangRongLin](https://github.com/XiangRongLin) for this feature.
```

Comment:
```suggestion
YouTube mixes are a feature now, and they are dropping in hot (and with a lot of bass). Opening a mix in previous versions resulted in the first video of the mix playing, nothing else. Now you see the full playlist and even a nice radio icon as an indicator. Enjoy listening to never ending lists and thank [@Stypox](Stypox) as well as [@XiangRongLin](https://github.com/XiangRongLin) for this feature.
```
Also, does `[@Stypox](Stypox)` actually work?

Reply:
No it does not my regex and me had a disagreement it seems.